scripts/hyperspectral/hyperspectral_workflow.sh

```diff
@@ -517,13 +518,13 @@ for ((fl_idx=0;fl_idx<${fl_nbr};fl_idx++)); do
 	typ_in_ENVI=$(grep '^data type' ${hdr_fl} | cut -d ' ' -f 4 | tr -d '\015')
 	xps_tm=$(grep 'current setting exposure' ${mtd_fl} | cut -d ':' -f 2 | tr -d '" ,\015' )
 	fl_clb="${drc_spt}/calibration_vnir_${xps_tm}ms.nc"
-	if [ "${wvl_nbr}" -eq 272 ]; then 
+	if [ "${wvl_nbr}" -eq 272 ] || [ "${wvl_nbr}" -eq 273 ]; then 
```

Comment:
Sorry, I meant 'metadata' more generally than just the metadata.json file. I was also referring to any available information that was available programmatically (the header file, calibration file, etc) as well as additional information that is available', e.g. in the header file, calibration file, or similar.

But for this particular line wvl_nbr is being used to define the sensor type - isn't that provided in the metadata? Again, not that this needs to be changed now, just trying to think about generalization.